src/components/cw20-token/services/cw20-token.service.ts

```diff
@@ -101,15 +99,6 @@ export class Cw20TokenService {
       );
     }
 
-    let limit = request.limit;
-    let offset = request.offset;
-
-    if (request.offset === 0) {
-      limit -= result.length;
-    } else {
-      offset -= result.length;
-    }
-
```

Comment:
sao chỗ này xóa đi a nhỉ

Reply:
chỗ này trả về toàn bộ data rồi trên FE sẽ thực hiện pagination ấy em.